parent/pom.xml

```diff
@@ -388,6 +388,8 @@
         <opensearch-testcontainers-version>2.1.0</opensearch-testcontainers-version>
         <openstack4j-version>3.12</openstack4j-version>
         <opentelemetry-version>1.41.0</opentelemetry-version>
+        <opentelemetry-log4j2-version>2.7.0-alpha</opentelemetry-log4j2-version>
```

Comment:
Hmm this is alpha software so we may have trouble maintaining it since this code change change a lot as it would not be stable yet.

Reply:
All versions available for this component are marked as `alpha`, unfortunately. This is used for testing only.

Reply:
okay are they released regulay ie how old is those versions you use now ?